src/Exceptions/InvalidMethod.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Exceptions;
+
+use Exception;
+
+/**
+ * @internal
+ */
+final class InvalidMethod extends Exception
```

Comment:
This is only used in one place, and I don't think we'll ever need it again, or I'm wrong? Shouldn't we be worried about the amount of new, in theory, unnecessary files?

Reply:
I think it's better for readability and testing, isn't it?